.serena/memories/project_overview.md

```
@@ -0,0 +1,32 @@
+# FLOSSIOULLK Project Overview
+
+## Purpose
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)